File: packages/acebit-nft-py/src/metadata.js
let fs = require("fs");
let axios = require("axios");

let ipfsArray = [];

for (let i = 1; i < 5; i++) {
    let paddedHex = ("0000000000000000000000000000000000000000000000000000000000000000" + i.toString(16)) //.substr("-64");
    ipfsArray.push({
        path: `metadata/${paddedHex}.json`,
        content: {
            image: `ipfs://QmX1xfbKoUEHtnoiVY9uEtH3X6YuJx2yzPg6ggp3zrFUzz/images/${paddedHex}.png`,
            name: `Ace NFT #${i}`,
            description: "Ace NFT Collection",
            properties: {
                "name": "Ace SUITE",
                "suite": "SUITE"
            }
        }
    })
}
axios.post("https://deep-index.moralis.io/api/v2/ipfs/uploadFolder", 
    ipfsArray,
    {
        headers: {
            "X-API-KEY": 'wnFbJ8XSeI8Ym4uOBsBHQfoYSOXF31OxAxs6ohXchHo7D7qvoJ66Qetxll5jrA46',
            "Content-Type": "application/json",
            "accept": "application/json"
        }
    }
).then( (res) => {
    console.log(res.data);
})
.catch ( (error) => {
    console.log(error)
})

File: packages/acebit-nft-py/metadata.py
import base64
import requests
from suite_distributor import SuiteDistributor
import helpers


BATCH = helpers.get_config()["BATCH"]


def main():
    _ipfsArray = []
    _ipfsImages = "ipfs://QmW4pkpwW8Lrv6etHLTbs6dc1Mjk7s2DSty4XnXTMKnkTU"

    suiteDistributor = SuiteDistributor()

    for _i in range(1, BATCH + 1):
        _suite = suiteDistributor.getSuite()

        _hex = hex(_i)
        _paddedHex = "0000000000000000000000000000000000000000000000000000000000000000{}".format(
            _hex[2:])
        _paddedHex = _paddedHex[-64:]

        print(_paddedHex)

        _meta = {
            "path": "metadata/{}.json".format(_paddedHex),
            "content": {
                "image": "{}/images/{}.png".format(_ipfsImages, _suite),
                "name": "Ace NFT #{}".format(_i),
                "description": "Ace NFT Collection",
                "properties": {
                    "name": "Ace {}".format(_suite),
                    "suite": "{}".format(_suite)
                }
            }


        }

        _ipfsArray.append(_meta)

    # print(_ipfsArray)

    headers = {
        "X-API-KEY": 'jhm4AZUaFMTlqcD5eHT2tuyY06pspE6Pu7S9eLo74XyQsPrmEhEP3ZQNrGKD8psh',
        "Content-Type": "application/json",
        "accept": "application/json"
    }
    URI = "https://deep-index.moralis.io/api/v2/ipfs/uploadFolder"

    # for _file in _ipfs_array:
    _res = requests.post(URI, headers=headers, json=_ipfsArray)
    print(_res.content)


if __name__ == '__main__':
    main()
